src/framework/theme/components/calendar-kit/components/calendar-year-picker/calendar-year-picker.component.ts

```diff
@@ -62,37 +60,23 @@ export class NbCalendarYearPickerComponent<D> implements OnChanges {
 
   @Output() yearChange = new EventEmitter<D>();
 
-  @HostBinding('class.medium')
-  get medium() {
-    return this.size === NbCalendarSize.MEDIUM;
-  }
-
-  @HostBinding('class.large')
+  @HostBinding('class.size-large')
   get large() {
     return this.size === NbCalendarSize.LARGE;
   }
 
   years: D[][];
 
-  constructor(protected dateService: NbDateService<D>) {
-  }
+  constructor(
```

Comment:
I'm not sure, but I believe we always format constructors another way:
```
  constructor(protected dateService: NbDateService<D>,
              protected yearModelService: NbCalendarYearModelService<D>) {
  }
```
BTW, I'm pretty sure it fits in one line

Reply:
See the answer here https://github.com/akveo/nebular/pull/2120#discussion_r355361367.

> I'm pretty sure it fits in one line

It is barely (ends on 117), so I found it more readable to place params on the separate lines.